src/js/tabs/logging.js

```diff
@@ -102,6 +102,11 @@ TABS.logging.initialize = function (callback) {
         ConfigStorage.get('logging_file_entry', function (result) {
             if (result.logging_file_entry) {
                 chrome.fileSystem.restoreEntry(result.logging_file_entry, function (entry) {
+                    if (chrome.runtime.lastError) {
+                        console.error(chrome.runtime.lastError.message);
```

Comment:
What happens if you just do `console.error(chrome.runtime.lastError)`? NW.js should be smart enough to dump the error message and stack trace in this case.

Reply:
This prints Object instead of the message property. Caller or Callee properties are not accessible in strict mode.
Was looking for a utility or common file to put this kind of stuff to de-deduplicate like in `serial.js` but then with an extra line: 
```
console.log((new Error).stack);
```